scripts/album.js

```diff
@@ -308,6 +311,15 @@ var togglePlayFromPlayerBar = function() {
     }
 };
 
+// Assignment #1
+var setCurrentTimeInPlayerBar = function(currentTime) {
+    $(this).attr('.current-time', 'timeupdate');
```

Comment:
This is the right idea, but you'll want to use your `filterTimeCode` function here.